pvacseq/lib/main.py

```diff
@@ -31,7 +31,7 @@ def main(args_input = sys.argv[1:]):
     parser.add_argument("output_dir",
                         help="Output directory for writing all result files"
                         )
-    parser.add_argument("-l", "--variant-peptide-length",
+    parser.add_argument("-l", "--peptide-sequence-length",
```

Comment:
Since you're changing the variable to peptide-sequence-length you can remove the dest argument.  Same with minimum-fold-change